ui/src/stylesheets/common.css

```
@@ -334,14 +334,14 @@ main {
     background: rgb(var(--v-theme-background));
     color: rgb(var(--v-theme-on-background));
 }
-
+/*
 .v-slider-track__background,
 .v-slider-track__fill,
 .v-slider-track__tick,
 .v-slider-thumb__surface {
     background-color: rgb(var(--v-theme-primary));
 }
```

Just look at this in more detail, the reason the thumb wasn't overriding is because we were setting `background-color` on the thumb, but Vuetify actually sets the `color` field with it's `color` setting. As such, the fix is:

```suggestion
.v-slider-track__background,
.v-slider-track__fill,
.v-slider-track__tick {
    background-color: rgb(var(--v-theme-primary));
}

.v-slider-thumb__surface {
    background-color: rgb(var(--v-theme-primary));
}
```

this way, we're overriding correctly, adn when the `thumb-color` property is then set, it overrides our CSS